app/components/publishers/vacancies_component.rb

```diff
@@ -18,8 +18,24 @@ def render?
     @organisation.all_vacancies.active.any?
   end
 
-  def selected_class(vacancy_type)
-    "govuk-tabs__list-item--selected" if @selected_type == vacancy_type
+  def selected(vacancy_type)
+    @selected_type == vacancy_type
+  end
+
+  def sortable
+    @vacancies.count > 1 && Organisation::JOB_SORTING_OPTIONS[@selected_type]&.count
```

Comment:
What's the second part of this line trying to check?

Reply:
there is no sorting for awaiting feedback page and hence there are no options for this selected_type

Reply:
have changed this to be a `.present?` check